Sources/FeaturevisorSDK/Evaluation+Dictionary.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Evaluation+Dictionary.swift
+//  
+//
+//  Created by Patryk Piwowarczyk on 28/09/2023.
+//
+
+import Foundation
+
+extension Evaluation {
+
+    func toDictionary(_ evaluation: Evaluation) -> [String: Any] {
```

Comment:
This is an extension to Evaluation so you don't need to pass the same object as a `parameter.`

Also, do this in that way:
```
  func toDictionary() -> [String: Any] {
    guard let data = try? JSONEncoder().encode(self) else { return [:] }
    return (try? JSONSerialization.jsonObject(with: data, options: .allowFragments)).flatMap { $0 as? [String: Any] ?? [:]}
  }
```